ui/server/src/test/scala/pl/touk/nussknacker/ui/process/DBProcessServiceSpec.scala

```diff
@@ -0,0 +1,75 @@
+package pl.touk.nussknacker.ui.process
+
+import akka.actor.ActorSystem
+import akka.testkit.TestProbe
+import com.typesafe.config.{Config, ConfigFactory}
+import org.scalatest.{FlatSpec, Matchers}
+import pl.touk.nussknacker.engine.api.deployment.ProcessActionType.Deploy
+import pl.touk.nussknacker.test.PatientScalaFutures
+import pl.touk.nussknacker.ui.api.helpers.TestFactory.processResolving
+import pl.touk.nussknacker.ui.api.helpers.{MockFetchingProcessRepository, TestFactory}
+import pl.touk.nussknacker.ui.db.DbConfig
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+import pl.touk.nussknacker.ui.util.ConfigWithScalaVersion
+import slick.jdbc.{HsqldbProfile, JdbcBackend}
+
+import java.time.Duration
+
+class DBProcessServiceSpec extends FlatSpec with Matchers with PatientScalaFutures {
+  import org.scalatest.prop.TableDrivenPropertyChecks._
+  import pl.touk.nussknacker.ui.api.helpers.TestCategories._
+  import pl.touk.nussknacker.ui.api.helpers.TestProcessUtil._
+
+  import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
can we move those to the top-level imports?

Reply:
Moved